gradle/versions.gradle

```diff
@@ -13,7 +13,7 @@
 
 dependencyManagement {
   dependencies {
-    dependency 'com.fasterxml.jackson.core:jackson-databind:2.11.2'
+    dependency 'com.fasterxml.jackson.core:jackson-databind:2.12.5'
     dependency 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.11.2'
```

Comment:
does `com.fasterxml.jackson.dataformat:jackson-dataformat-yaml` version required to be upgraded as well?

Reply:
Would be good to use a dependency set here too to keep the jackson versions in sync

Reply:
@jframe  They are two different groups unfortunately.

Reply:
AFAIK, the azure upgrade doesn't require dataformat-yaml to be upgraded (signers doesn't use it) Updating dataformat-yaml causes CommandlineParserTest failures which is starting to feel out of scope for this PR so I'm going to leave it as 2.11.2 unless there's any objections. I'd rather handle it in a 'upgrade web3signers deps' PR.

Reply:
👍🏼 .. lets keep it as it is then.